### ISSUE TITLE
When --ram is set to any value, 'Link' fails with File Not Found

### ISSUE THREAD

**Issue by steventroughtonsmith** [ksherlock/mpw#17]
If you pass any value to mpw's --ram param, the 68k Linker seems to fail. This is relevant because some of MPW's own examples won't link with the 16M default cap.

Setting --ram=16M does not kill Link; any other value seems to.

```
### Link: File not found (OS error -43)  
### Link: Error: Resource I/O error, # (Error 6) opening/creating file.
### Link: Errors prevented normal completion.
```

Sample code available at https://gist.github.com/steventroughtonsmith/2b672ee795c86632ba83

**Comment by ksherlock** [comment]
Ahh, `StripAddress` blindly limits addresses to 24 bits/16 MB.

**Comment by ksherlock** [comment]
Fixed with b146399a7efa1b68bc1fb4204562e57597707785.
